### PR TITLE
fix: Correct erroneous information in some Establishment Classes

### PR DESCRIPTION
**Pull request by keytsai**
1. Correct TvStation's inheritance class
2. Correct CheeseFactory's CardType

### CODE REVIEW

**Review by shmily7829 — COMMENTED** [review]
電視台需留下TODO資訊

File: src/main/java/domain/card/establishment/TvStation.java [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+
+public class TvStation extends Establishment {
     public TvStation() {
-        super("電視台", 7, CardType.MAJOR_ESTABLISHMENT);
+        super("電視台", 7, CardType.MAJOR_ESTABLISHMENT, Set.of(6), IndustryColor.PURPLE);
+    }
+
+    @Override
+    protected void doTakeEffect(Game game) {
+
```
Comment (shmily7829) [inline src/main/java/domain/card/establishment/TvStation.java:15]: Please leave TODO descript

**Review by shmily7829 — CHANGES_REQUESTED** [review]
Review done

File: src/main/java/domain/card/establishment/TvStation.java (hunk 1 above)
Comment (keytsai) [reply src/main/java/domain/card/establishment/TvStation.java]: Done

**Review by shmily7829 — APPROVED** [review]
LGTM

**Pull request merged**